test/test_config.rb

```diff
@@ -45,6 +45,7 @@ def test_corrupt_settings
 
     config = nil
     captured = capture_stderr do
+      !!config # avoid warning
```

Comment:
This feels a little weird... what's the warning?

Reply:
```
discourse_theme/test/test_config.rb:46: warning: assigned but unused variable - config
```

ie. `nil` is assigned but never used. IN THEORY you could just not assign `config`, but I sometimes get the STDERR printed even though `capture_stderr` is supposed to capture and stop it from printing.

Like we saw in private chat, this seems to be a bug in `capture_stderr` related to random test ordering, but this is test code and not application code (which is correct in all cases since the tests always succeeds even with the incorrect STDERR printing).

Per your suggestion I'll remove the assignment altogether; we can fix the non deterministic printing later.